src/pages/manage/system/log.tsx

```diff
@@ -0,0 +1,98 @@
+import { Card, Text, Title } from '@mantine/core';
+import { useInterval } from '@mantine/hooks';
+import fs from 'fs';
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next';
+import { useTranslation } from 'next-i18next';
+import Head from 'next/head';
+import { useEffect, useState } from 'react';
+import { ManageLayout } from '~/components/layout/Templates/ManageLayout';
+import { getServerAuthSession } from '~/server/auth';
+import { logFilePath } from '~/tools/constants';
+import { getServerSideTranslations } from '~/tools/server/getServerSideTranslations';
+import { checkForSessionOrAskForLogin } from '~/tools/server/loginBuilder';
+import { boardNamespaces } from '~/tools/server/translation-namespaces';
+import { api } from '~/utils/api';
+
+const LogPage = ({ logFileExists }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const { t } = useTranslation('system/log');
+  const metaTitle = `${t('metaTitle')} • Homarr`;
+
+  const utils = api.useContext();
+  const { data } = api.log.poll.useQuery(undefined, {
+    retry: false
+  });
+
+  const refetchInterval = 5 * 1000;
+
+  const [milliseconds, setMilliseconds] = useState(0);
+  const interval = useInterval(
+    () =>
+      setMilliseconds((previous) => {
+        const newValue = previous + 100;
+        if (newValue > refetchInterval) {
+          void utils.log.poll.refetch();
+          return 0;
+        }
+        return newValue;
+      }),
+    100
+  );
+
+  useEffect(() => {
+    interval.start();
+    return interval.stop;
+  }, []);
+
+  return (
+    <ManageLayout>
+      <Title mb="lg">{t('title')}</Title>
+      <Head>
+        <title>{metaTitle}</title>
+      </Head>
+
+      <Card>
+        {!logFileExists && (
+          <Text color="dimmed" mb="md">{t('notSupported')}</Text>
+        )}
+        {data?.map((line) => (
```

Comment:
Use a proper loading state or load the initialData during SSR (see docker page for example)